docs/platform/reference/tic-faq.md

```diff
@@ -0,0 +1,40 @@
+<!-- omit in toc -->
+# TIC Frequently Asked Questions
```

Comment:
I see that the FAQ page format is different from [https://docs1.stg.tizen.org/staging/1564/platform/reference/gbs/gbs-faq/](https://docs1.stg.tizen.org/staging/1564/platform/reference/gbs/gbs-faq/) and MIC Faq, however I like this page's format so I guess we can keep it as is.